tests/e2e/support/commands.js

```diff
@@ -1,14 +1,70 @@
 import { getCardIds, hasValidSuitAndRank, cardsMatch, printCard } from './helpers';
 import { myUser, opponentOne, playerOne, playerTwo } from '../fixtures/userFixtures';
 
-const gameStateApiError = 'This action is not supported yet in GameState API';
-
 /**
  * Require & configure socket connection to server
  */
 const io = require('sails.io.js')(require('socket.io-client'));
 io.sails.url = 'localhost:1337';
 io.sails.useCORSRouteToGetCookie = false;
+const env = Cypress.env('gameStateAPI');
+
+const transformGameUrl = (api, slug) => {
+  if (env !== 'true') {
+    return `/api/${api}/${slug}`;
+  }
+  cy.window()
+    .its('cuttle.gameStore.id')
+    .then((gameId) => {
+      switch (slug) {
+        case 'draw':
+        case 'points':
+        case 'faceCard':
+        case 'scuttle':
+        case 'untargetedOneOff':
+        case 'targetedOneOff':
+        case 'jack':
+        case 'counter':
+        case 'resolve':
+        case 'resolveThree':
+        case 'resolveFour':
+        case 'resolveFive':
+        case 'seven/points':
+        case 'seven/scuttle':
+        case 'seven/faceCard':
+        case 'seven/jack':
+        case 'seven/untargetedOneOff':
+        case 'seven/targetedOneOff':
+        case 'pass':
+          // add all the move-making ones here
+          return `api/game/${gameId}/move/${slug}`;
+        default:
+          return `/api/${api}/${slug}`;
```

Comment:
Why does one have a preceding slash, but not the other?

Reply:
I think we probably want the preceding slash.

This is a copy paste from the game store so we should probably change it there too